### PR TITLE
Add a sidebar item for “interesting links”

### PR DESCRIPTION
**Pull request by alexwlchan**
Still needs documenting somewhere, brief review, and squashing.

### CODE REVIEW

File: src/project.rs [hunk 1]
```diff
@@ -57,6 +65,39 @@ fn get_str_field(obj: &Map<String, Value>, key: &str) -> String {
...
+    for link in result {
+        // Try and turn the link into an object, but if it doesn't work,
+        // skip the link.
+        let link_obj = match link.as_object() {
+            Some(v) => v,
+            None => continue,
+        };
+        let title = get_str_field(link_obj, "title");
+        let url = get_str_field(link_obj, "url");
+        if title != "" && url != "" {
```
Comment (alexwlchan) [inline src/project.rs]: Missing a test case: if one or both of the title/url fields is an empty string, we skip the link.
